### DISCUSSION TITLE
Automatically updating screenshots

### DISCUSSION THREAD

**Discussion by sircharlo** [sircharlo/meeting-media-manager#1136] [Ideas] [answered]
Could we look into integrating the [screenshot](https://playwright.dev/docs/screenshots) feature of Playwright to automatically update the documentation screenshots at every release? Would that even be feasible? @mtdvlpr

**Comment by mtdvlpr** [comment]
I definitely see possibilities in combination with this:
https://stackoverflow.com/a/71302017

**Comment by mtdvlpr** [comment, marked as answer]
@sircharlo, I've got some [automatically created screenshots](https://github.com/mtdvlpr/meeting-media-manager/tree/docs/assets/test/screenshots) for you to review! The missing images can't be automatically updated (can't take picture of both main and media window and can't upload custom files in test). I have a working script ready that we can run in the test job every time a new version is released.

**Reply by mtdvlpr** [reply]
You'll need to [create a personal access token](https://github.com/settings/tokens/new) that won't expire and has everything from the `repo` scope selected. It needs to be stored in the repository secrets as `PERSONAL_TOKEN` (or something else if you prefer).

**Reply by sircharlo** [reply]
Awesome!! `meeting.png` and `les32.jpg` don't seem to have worked properly, but otherwise this is great. Maybe we can simply replace or remove the impossible screenshots with other ones? (For example, a screenshot of only the media window, etc.)
The goal being to automate as much as possible, of course.

**Reply by mtdvlpr** [reply]
Agreed. `les32.jpg` was manually added to see what would happen to images already present in the folder. They stay, so that's good, because the images that aren't automated should not be removed suddenly. `meeting.png` was indeed wrong. I'll see what I can do about the remaining images and make a PR.

**Reply by sircharlo** [reply]
> You'll need to [create a personal access token](https://github.com/settings/tokens/new) that won't expire and has everything from the `repo` scope selected. It needs to be stored in the repository secrets as `PERSONAL_TOKEN` (or something else if you prefer).

Done.
